src/components/AccountSettings/ClientProfilePage.tsx

```diff
@@ -510,6 +528,24 @@ class ClientProfilePage extends Component<Props, State> {
                   </div>
                   <div>
                     <div className="text-center mx-4">
+                      <div className="btn-group mt-2">
+                        <button
+                          className="btn w-25 btn-outline-primary"
+                          type="button"
+                          onClick={this.zoomIn}
+                          disabled={this.state.zoom >= 2}
+                        >
+                          +
```

Comment:
Another nit-picky comment lol. Is there a figma for this functionality? Because I feel like I usually see the zoom in button on the right side of the zoom out one.